src/where.c

```diff
@@ -83,6 +83,22 @@ static int dcc_lock_one(struct dcc_hostdef *hostlist,
                         int *cpu_lock_fd);
 
 
+void dcc_read_localslots_configuration()
```

Comment:
Is the purpose of this to check for syntax errors in the host list? Because the code doesn't seem to do anything.

Reply:
oh, nevermind

Reply:
ahh I see in dcc_parse_hosts()